examples/createContract/src/spec.md

```diff
@@ -19,11 +21,18 @@ storage
 if
 
     #newAddr(ACCT_ID, Nonce_Mom) == KID
+    Id == keccakIntList(                                                                                                 \
+        keccak(#parseByteStackRaw("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)")) \
+        keccak(#parseByteStackRaw("Kid V1"))                                                                             \
+        keccak(#parseByteStackRaw("1"))                                                                                  \
+        VChainId                                                                                                         \
+        KID                                                                                                              \
+    )
```

Comment:
could this go in a `where` block?

Reply:
Oh yes probably - now that we have that technology available :-)